_format/1.1/index.md

```diff
@@ -504,18 +513,22 @@ For example:
 ### <a href="#document-links" id="document-links" class="headerlink"></a> Links
 
 Where specified, a `links` member can be used to represent links. The value
-of each `links` member **MUST** be an object (a "links object").
+of this member **MUST** be an object (a "links object").
 
-Each member of a links object is a "link". A link **MUST** be represented as
-either:
+<a href="#document-links-link" id="document-links-link"></a>
+Within this object, a link **MUST** be represented as either:
```

Comment:
The prior language equated each member with a single link. The removal of the phrase creates the same issue I'm trying to resolve for relationships in #946. Another similarity to #946 is that the names of the links are not mentioned at all.